src/components/comparison/comparison-side/comparison-side.tsx

```diff
@@ -429,6 +428,7 @@ export const ComparisonSide = ({
                 memoryStats={memoryStats}
                 tilesetStats={tilesetStats}
                 loadingTime={loadingTime}
+                updateNumber={updateStatsNumber}
```

Comment:
Do we really need this number or can we just use `useForceUpdate` hook ?

Reply:
It doesn't work here for some reason